servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/AbstractAsynchronousPublisherOperator.java

```diff
@@ -35,11 +35,15 @@
 
     private final Publisher<T> original;
 
-    AbstractAsynchronousPublisherOperator(Publisher<T> original, Executor executor) {
-        super(executor);
+    AbstractAsynchronousPublisherOperator(Publisher<T> original) {
         this.original = requireNonNull(original);
     }
 
+    @Override
+    Executor executor() {
```

Comment:
(for visibility): the plan is to eventually remove the `executor()` methods in subsequent PRs, correct?

Reply:
Correct.